server/matchmaker/pop_timer.py

```diff
@@ -0,0 +1,68 @@
+import asyncio
+from collections import deque
+from time import time
+from typing import Callable, Deque
+
+import server
+
+from .. import config
+from ..decorators import with_logger
+
+
+@with_logger
+class PopTimer(object):
```

Comment:
It would be cool to have docstring, to not to read what that class is trying to solve, but just read it, why it's here.